src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -173,8 +173,7 @@ private async Task ProcessContainerChange(WatchEventType watchEventType, Contain
 
         var objectChangeType = ToObjectChangeType(watchEventType);
 
-        var containerViewModel = ConvertToContainerViewModel(
```

Comment:
Why were these calls changed? Static methods were intentional. There is no concurrency control in those methods.

Reply:
I needed to access another instance field (IOutgoingPeerResolver). The static methods already had a high number of parameters, and they always used the same values:

* _applicationModel
* _servicesMap.Values
* _endpointsMap.Values

These weren't being copied before they're passed to the static methods. Why was it better that they were static?